lib/Data/Validate/Sanctions.pm

```diff
@@ -1,54 +1,84 @@
 package Data::Validate::Sanctions;
 
 use strict;
-use 5.008_005;
-our $VERSION = '0.04';
+our $VERSION = '0.05';
 
 require Exporter;
-our @ISA = qw(Exporter);
-our @EXPORT_OK = qw/is_sanctioned/;
+our @ISA       = qw(Exporter);
```

Comment:
it is enough to have `use Exporter qw/import/;` instead of inheritance, or @tfoertsch will not merge it  :)